css/index.css

```diff
@@ -0,0 +1,109 @@
+body{
+    font-family: Arial, Helvetica, sans-serif;
+    padding: 0 16px;
+}
+nav{
+    text-align: right;
+}
+.menu-item{
+    font-size: 13px;
+    color: rgba(0,0,0,0.87);
+    text-decoration: none;
+    position: relative;
+    top: -8px;
+    margin-right: 16px;
+
+}
+.menu-item:hover{
+    text-decoration: underline;
+}
+#profile-pic{
```

Comment:
Kindly add the circle ring around the picture shown below
![Screen Shot 2021-02-16 at 19 25 11](https://user-images.githubusercontent.com/12280033/108098988-b76c6a00-708c-11eb-824b-3dde56fd660e.png)